Source: kde bug tracking system
Upstream: not yet in ktorrent, but probably something very similar will be
Reason: cmake fails, the issue is the same as
        https://bugs.kde.org/show_bug.cgi?id=288712

diff -rupN original/CMakeLists.txt new/CMakeLists.txt
--- original/CMakeLists.txt	2011-11-22 17:36:26.000000000 +0000
+++ new/CMakeLists.txt	2011-12-23 02:26:50.438469357 +0000
@@ -2,6 +2,7 @@ project(KTORRENT)
 find_package(Qt4 4.6.0 REQUIRED)
 find_package(KDE4 4.4.0 REQUIRED)
 find_package(KTorrent REQUIRED)
+set (Boost_NO_BOOST_CMAKE TRUE)
 find_package(Boost REQUIRED)
 
 # set some default settings
